homeassistant/components/bluetooth/__init__.py

```diff
@@ -251,6 +286,8 @@ def _async_trigger_discovery() -> None:
         EVENT_HOMEASSISTANT_STOP, hass_callback(lambda event: cancel())
     )
 
+    await _async_check_haos_version(hass)
```

Comment:
Can we limit this to when a USB adapter is being used?

Reply:
It already is since it should return early if there are no config entries present and config entry ==
adapter in this case.  But could use a test for no adapters present.

Reply:
I missed that check. All good